localstack/config.py

```diff
@@ -668,11 +668,6 @@ def in_docker():
 # truncate output string slices value
 LAMBDA_TRUNCATE_STDOUT = int(os.getenv("LAMBDA_TRUNCATE_STDOUT") or 2000)
 
-# A comma-delimited string of stream names and its corresponding shard count to
-# initialize during startup.
-# For example: "my-first-stream:1,my-other-stream:2,my-last-stream:1"
-KINESIS_INITIALIZE_STREAMS = os.environ.get("KINESIS_INITIALIZE_STREAMS", "").strip()
-
```

Comment:
This only made sense when a single instance of Kinesis Mock was running. Removed to avoid conflicts.

Reply:
That's a good point. I've restored the config variable for now, which will create the streams for all accocunts. However, I think we should make use of the v2.0 jump to remove it. IMO there should be a single way to create resources i.e. via LocalStack API.